client/src/components/settings/Username.js

```diff
@@ -52,6 +52,9 @@ const invlaidCharError = {
 const valididationSuccess = { valid: true, error: null };
 const usernameTooShort = { valid: false, error: 'Username is too short' };
 
+const hex = '[0-9a-f]';
+const tempUserRegex = new RegExp(`^fcc${hex}{8}-\(${hex}{4}-\){3}${hex}{12}$`);
```

Comment:
```suggestion
const tempUserRegex = new RegExp(`^fcc${hex}{8}-(${hex}{4}-){3}${hex}{12}$`);
```
Single escapes don't seem to change anything, so they might as well go.